cSpell.json

```diff
@@ -1,188 +1,188 @@
 {
-    "version": "0.2",
```

Comment:
Why was this reformatted? FWIW, I prefer 4 spaces since it's more similar to our Solidity spacing, and it feels less cramped. Was there a particular reason that we're using 2 spaces for Typescript formatting?

Reply:
not intentional, likely from prettier issues mentioned above, will restore